icubam/www/handlers/version.py

```diff
@@ -0,0 +1,36 @@
+import sys
+from subprocess import check_output
+from typing import Dict, List, Optional, Callable
+import traceback
+
+import tornado.web
+
+import icubam
+from icubam.www.handlers import base
+from icubam.www import updater
+
+
+class VersionHandler(base.BaseHandler):
+  ROUTE = '/version'
+
+  def initialize(self, config, db):
+    super().initialize(config, db)
+    self.updater = updater.Updater(self.config, self.db)
+
+  def get_data(self) -> List[Dict]:
+    """Get sanity checks data"""
+    data = {}
+    data['version'] = icubam.__version__
+    try:
+      git_hash = check_output(['git', 'rev-parse', '--short', 'HEAD'],
+                              encoding='utf-8').strip()
+    except Exception:
+      git_hash = 'NA'
+    data['git-hash'] = git_hash
+    bed_counts = self.db.get_bed_counts()
```

Comment:
Can we wrap this in a try/catch and specify in the returned data whether it returned correctly && len(bedcounts) >0, or incorrectly (len == 0 or Exception)?

Reply:
We then would need specific code to log the traceback + error message, and manually return HTTP 500 error status. See https://github.com/icubam/icubam/pull/113#issuecomment-607199272

Reply:
I would say let's do the simplest thing that can work (i.e. do not catch the exception)? We don't want to supply too much context anyway... My 2cents :)